### PR TITLE
Upgrade to cosmwasm 1.0.0, remove redundant contracts

### PR DESCRIPTION
**Pull request by SunshineRider**

### CODE REVIEW

File: contracts/proxy_to_vkr/Cargo.toml [hunk 1]
```diff
@@ -20,17 +20,16 @@ crate-type = ["cdylib", "rlib"]
...
+cosmwasm-std = "1.0"
+cw-storage-plus = "0.13.2"
 schemars = "0.8.1"
 serde = { version = "1.0.125", default-features = false, features = ["derive"] }
-thiserror = { version = "1.0.24" }
-cw2 = "0.8.0"
-cw20 = "0.8.0"
-valkyrie = { version = "1.0.7"}
-astroport = { version = "1.0.1" }
+thiserror = { version = "1.0.20" }
```
Comment (epanchee) [inline contracts/proxy_to_vkr/Cargo.toml]: Why downgrade?

File: contracts/proxy_to_vkr/Cargo.toml [hunk 2]
```diff
@@ -20,17 +20,16 @@ crate-type = ["cdylib", "rlib"]
...
+cw2 = "0.13.2"
+cw20 = "0.13.2"
+astroport = { git = "https://github.com/astroport-fi/astroport-core.git", branch = "release/terra2" }
+valkyrie = { git = "https://github.com/valkyrieprotocol/contracts.git", branch = "feature/migrate_terra2" }
```
Comment (epanchee) [inline contracts/proxy_to_vkr/Cargo.toml]: I'm not sure that we can rely on temporal branch in 3rd party repos. Are we sure that Valkyrie team won't change/remove this branch?
Anyway I would suggest to use rev="..." to pin exact commit hash.

File: contracts/proxy_to_vkr/src/contract.rs [hunk 3]
```diff
@@ -192,7 +192,7 @@ fn withdraw(
         funds: vec![],
         msg: to_binary(&ExecuteMsg::Callback(
             CallbackMsg::TransferLpTokensAfterWithdraw {
-                account,
+                account: deps.api.addr_validate(&account)?,
```
Comment (epanchee) [inline contracts/proxy_to_vkr/src/contract.rs]: Use astroport::asset::addr_validate_to_lower()

**Review by ShadoySV — APPROVED** [review]

File: contracts/proxy_to_vkr/Cargo.toml (hunk 1 above)
Comment (SunshineRider) [reply contracts/proxy_to_vkr/Cargo.toml]: corrected

File: contracts/proxy_to_vkr/Cargo.toml (hunk 2 above)
Comment (SunshineRider) [reply contracts/proxy_to_vkr/Cargo.toml]: updated to rev = "b5fcb666f17d7e291f40365756e50fc0d7b9bf54" which is the latest commit hash

File: contracts/proxy_to_vkr/src/contract.rs (hunk 3 above)
Comment (SunshineRider) [reply contracts/proxy_to_vkr/src/contract.rs]: resolved

**Review by epanchee — APPROVED** [review]

**Pull request merged**